### ISSUE TITLE
Deploying smart contract on Windows is failing

### ISSUE THREAD

**Issue by arup1234** [nebulasio/go-nebulas#99]
Hi, I haven't installed Nebulas yet because the instructions for Windows haven't been released yet. But, I was going through a similar issue here where someone commented that installation is required only for running a node, not for deploying a smart contract. Is this true? If so, why am I receiving a 400 (resource not found) error when trying to deploy a smart contract?
Thanks

**Comment by ChengOrangeJu** [comment]
@arup1234 Could you explain a little bit more about your operation? Have you deployed your smart contract on testnet?

**Comment by arup1234** [comment]
Sure, thanks for your response. I was trying to deploy the smart contract on testnet by following the tutorials provided. However, it fails. When I click on test after pasting my contract, it gives me the following result, {"result":"\"\"","execute_err":"insufficient balance","estimate_gas":"21815"}. However, I saw in the tutorial that even when one has insufficient balance, they're able to click on submit and it generates a transaction hash. When I try to submit, in the submit result, both txhash and contract_address remain unpopulated and the network shows that https://testnet.nebulas.io/v1/user/rawtransaction sends a status code of 400 with the response showing {"error":"insufficient balance"}. I have attached a screenshot for your reference here. So, do I really need to have NAS to deploy a smart contract? If so, is there any way to get free NAS while on testnet for Nebulas?

Thanks


![image](https://user-images.githubusercontent.com/39038123/39707007-5d79cdfe-51d0-11e8-9a6f-c19cd4c45416.png)
